pkg/engine/ipaddresses.go

```diff
@@ -30,6 +30,25 @@ func CreatePublicIPAddress() PublicIPAddressARM {
 	}
 }
 
+func CreatePublicIPAddressForSLB() PublicIPAddressARM {
```

Comment:
this function seems very similar to the one above (except for the DNSSettings), maybe we can reuse that code and just add the DNSSettings for the other scenario